proposals/custom-descriptors/Overview.md

```diff
@@ -536,7 +536,115 @@ console.log(counter.get()); // 1
 
 ## Declarative Prototype Initialization
 
-TODO
+We expect toolchains to need to configure thousands of JS prototypes and tens of thousands of methods,
+so to reduce startup latency we want a declarative method of constructing, importing,
+and populating the `DescriptorOptions` objects.
+
+The declarative API must support these features:
+
+ - Constructing new prototype objects
+ - Using existing prototype objects provided at instantiation time
+ - Creating prototype chains
+ - Synthesizing and attaching methods (including getters and setters) to prototypes
```

Comment:
Could there be a goal:
* Polyfillable using the existing JS API by generating JS glue code that uses the existing JS API

?